examples/plot_split_dataset.py

```diff
@@ -0,0 +1,58 @@
+"""Split Dataset Example
+========================
+
+In this example, we show multiple ways of how to split datasets.
+"""
+
+# Authors: Lukas Gemein <l.gemein@gmail.com>
+#
+# License: BSD (3-clause)
+
+from IPython.display import display
```

Comment:
why do you need this?
adding a dependency on ipython to do a print seems an overkill to me.

Reply:
I just copied it from examples.plot_dataset_example.py, so the dependency was already there. Could of course remove it everywhere.